lib/Build.js

```diff
@@ -0,0 +1,115 @@
+var events = require('events')
```

Comment:
We should add `"use strict";` for all new files so we can use `let` and other ES6 goodness.